kong/api/routes/cluster.lua

```diff
@@ -1,57 +1,125 @@
 local singletons = require "kong.singletons"
 local responses = require "kong.tools.responses"
+local cache = require "kong.tools.database_cache"
+local event_types = require("kong.core.events").TYPES
 
 local pairs = pairs
 local table_insert = table.insert
 local string_upper = string.upper
 
+local OTHER = "other"
+
+local function get_members(filters)
+  local members, err = singletons.serf:members()
+  if err then
+    return nil, err
+  end
+
+  local result = {}
+  for _, member in pairs(members) do
+    local matches = true
+    if filters then
+      for k, v in pairs(filters) do
+        matches = matches and member[k] == v
+      end
+    end
+    if matches then
+      table_insert(result, {
+        name = member.name,
+        address = member.addr,
+        status = member.status
+      })
+    end
+  end
+
+  return result
+end
+
+local function increment(key, value)
+  local ok, err = cache.incr(key, value)
```

Comment:
Still the same anti-pattern of using the database caching module (with no other benefit than a simple `dict:incr()` wrap) to perform other operations (here incrementing a counter of events).